applications/GeoMechanicsApplication/tests/line_interface_elements/Neumann_multi_stage_reset_displacements/ProjectParameters_stage2.json

```diff
@@ -0,0 +1,158 @@
+                        "skin_output":         false,
+                        "plane_output":        [],
+                        "nodal_results":       ["DISPLACEMENT"],
+                        "gauss_point_results": []
+                    },
+                    "point_data_configuration":  []
+                }
+            }
+        }]
+    },
+    "processes": {
+        "constraints_process_list": [{
+            "python_module": "apply_vector_constraint_table_process",
+            "kratos_module": "KratosMultiphysics.GeoMechanicsApplication",
+            "process_name":  "ApplyVectorConstraintTableProcess",
+            "Parameters":    {
+                "model_part_name": "PorousDomain.FixedNodes",
+                "variable_name":   "DISPLACEMENT",
+                "active":          [true,true,false],
+                "is_fixed":        [true,true,true],
+                "value":           [0.0,0.0,0.0],
+                "table":           [0,0,0]
+            }
+        },{
+            "python_module": "apply_scalar_constraint_table_process",
+            "kratos_module": "KratosMultiphysics.GeoMechanicsApplication",
+            "process_name":  "ApplyScalarConstraintTableProcess",
+            "Parameters":    {
+                "model_part_name": "PorousDomain.AllNodes",
+                "variable_name":   "WATER_PRESSURE",
+                "active":          true,
+                "is_fixed":        true,
+                "value":           0.0,
+                "table":           0
+            }
+        },{
+            "python_module" : "reset_displacement_process",
+            "kratos_module" : "KratosMultiphysics.GeoMechanicsApplication",
+            "process_name"  : "ResetDisplacementProcess",
+            "Parameters"    : {
+                "model_part_name": "PorousDomain.Interfaces"
+            }
+        }],
+        "loads_process_list": [{
+            "python_module": "apply_normal_load_table_process",
+            "kratos_module": "KratosMultiphysics.GeoMechanicsApplication",
+            "process_name":  "ApplyNormalLoadTableProcess",
+            "Parameters":    {
+                "model_part_name": "PorousDomain.LoadedSide",
+                "variable_name":   "NORMAL_CONTACT_STRESS",
+                "active":          [true,false],
+                "value":           [0.0,0.0],
+                "table":           [2,0],
+                "fluid_pressure_type": "Uniform"
+            }
+        },{
+            "python_module": "apply_normal_load_table_process",
+            "kratos_module": "KratosMultiphysics.GeoMechanicsApplication",
+            "process_name":  "ApplyNormalLoadTableProcess",
+            "Parameters":    {
+                "model_part_name": "PorousDomain.LoadedSide",
+                "variable_name":   "TANGENTIAL_CONTACT_STRESS",
+                "active":          [false,true],
+                "value":           [0.0,667.0],
```

Comment:
For the normal contact stress the value array is filled with zeros, values come from the table. Here a value is given , but it will be taken from the table anyway. The syntax itself is already confusing, may leave the unused value at zero.

Reply:
Yes, done (and in the test I based this one on)